src/js/stores/__tests__/NodeHealthStore-test.js

```diff
@@ -65,14 +65,14 @@ describe('NodeHealthStore', function () {
     });
 
     it('dispatches the correct event upon success', function () {
-      var mockedFn = jest.genMockFunction();
+      var mockedFn = jasmine.createSpy();
       NodeHealthStore.addChangeListener(EventTypes.HEALTH_NODES_CHANGE, mockedFn);
       AppDispatcher.handleServerAction({
         type: ActionTypes.REQUEST_HEALTH_NODES_SUCCESS,
         data: []
       });
 
-      expect(mockedFn.mock.calls.length).toEqual(2);
+      expect(mockedFn.calls.count()).toEqual(1);
```

Comment:
just looked at the code. It should be two because when you `addChangeListener` and `shouldPoll` is true, it makes a call and in your `beforeEach` it instantly calls the success function. so that's call number 1.

call number 2 is when you do `handleServerAction` on line 70.

this test is passing because `shouldPoll` is incorrect atm